models/reporter.go

```diff
@@ -79,7 +79,7 @@ SUM(num_queries) AS num_queries,
 {{ end }}
 rowNumberInAllBlocks() AS total_rows
 FROM metrics
-WHERE period_start > :period_start_from AND period_start < :period_start_to
+WHERE period_start >= :period_start_from AND period_start <= :period_start_to
```

Comment:
Do we really need both inclusive ranges for `period_start` field?

Reply:
That's a good question, we have 4 queries in QAN, but only this one doesn't use inclusive ranges, that's why we are having different results for the List and Details.